### ISSUE TITLE
add paper "Cross-Lingual Question Answering over Knowledge Base as Reading Comprehension"

### ISSUE THREAD

**Issue by xixi019** [KGQA/leaderboard#117]
https://arxiv.org/pdf/2302.13241.pdf

**Comment by LongquanJiang** [comment]
A new subset of WebQSP dataset, WebQSP-zh, was constructed for cross-lingual KGQA task, should we also put it into our leaderboard?

**Comment by xixi019** [comment]
I do not think so, since there's nothing much new about it. What do you think? @RicardoUsbeck

**Comment by RicardoUsbeck** [comment]
Well, maybe make a footnote on the WebQSP page that a ZH version exists.
